packages/truffle-core/lib/testing/testsource.js

```diff
@@ -42,52 +48,199 @@ TestSource.prototype.resolve = function(import_path, callback) {
-    });
+  if (import_path === "truffle/AssertBytes32Array.sol") {
+    return fs.readFile(
+      path.resolve(path.join(__dirname, "AssertBytes32Array.sol")),
+      { encoding: "utf8" },
+      function(err, body) {
+        callback(err, body, import_path);
+      }
+    );
+  }
+
+  if (import_path === "truffle/AssertAddress.sol") {
+    return fs.readFile(
+      path.resolve(path.join(__dirname, "AssertAddress.sol")),
+      { encoding: "utf8" },
+      function(err, body) {
+        callback(err, body, import_path);
+      }
+    );
+  }
+
+  if (import_path === "truffle/AssertAddressArray.sol") {
+    return fs.readFile(
+      path.resolve(path.join(__dirname, "AssertAddressArray.sol")),
+      { encoding: "utf8" },
+      function(err, body) {
+        callback(err, body, import_path);
+      }
+    );
+  }
+
+  if (import_path === "truffle/AssertAddressPayableArray.sol") {
+    return fs.readFile(
+      path.resolve(path.join(__dirname, "AssertAddressPayableArray.sol")),
+      { encoding: "utf8" },
+      function(err, body) {
+        callback(err, body, import_path);
+      }
+    );
+  }
+
+  if (import_path === "truffle/AssertBool.sol") {
+    return fs.readFile(
+      path.resolve(path.join(__dirname, "AssertBool.sol")),
+      { encoding: "utf8" },
+      function(err, body) {
+        callback(err, body, import_path);
+      }
+    );
+  }
+
+  if (import_path === "truffle/AssertUint.sol") {
+    return fs.readFile(
+      path.resolve(path.join(__dirname, "AssertUint.sol")),
+      { encoding: "utf8" },
+      function(err, body) {
+        callback(err, body, import_path);
+      }
+    );
+  }
+
+  if (import_path === "truffle/AssertUintArray.sol") {
+    return fs.readFile(
+      path.resolve(path.join(__dirname, "AssertUintArray.sol")),
+      { encoding: "utf8" },
+      function(err, body) {
+        callback(err, body, import_path);
+      }
+    );
+  }
+
+  if (import_path === "truffle/AssertInt.sol") {
+    return fs.readFile(
+      path.resolve(path.join(__dirname, "AssertInt.sol")),
+      { encoding: "utf8" },
+      function(err, body) {
+        callback(err, body, import_path);
+      }
+    );
+  }
+
+  if (import_path === "truffle/AssertIntArray.sol") {
+    return fs.readFile(
+      path.resolve(path.join(__dirname, "AssertIntArray.sol")),
+      { encoding: "utf8" },
+      function(err, body) {
+        callback(err, body, import_path);
+      }
+    );
+  }
+
+  if (import_path === "truffle/AssertBalance.sol") {
+    return fs.readFile(
+      path.resolve(path.join(__dirname, "AssertBalance.sol")),
+      { encoding: "utf8" },
+      function(err, body) {
+        callback(err, body, import_path);
+      }
+    );
+  }
+
+  if (import_path === "truffle/AssertGeneral.sol") {
+    return fs.readFile(
+      path.resolve(path.join(__dirname, "AssertGeneral.sol")),
+      { encoding: "utf8" },
+      function(err, body) {
+        callback(err, body, import_path);
+      }
+    );
```

Comment:
This is another bunch of code that could use to be replaced with a list of library names + for loop.